link-parent/link-ilp-over-http/src/main/java/org/interledger/link/http/IncomingLinkSettings.java

```diff
@@ -75,38 +75,47 @@
                   .map(Object::toString)
                   .map(String::toUpperCase)
                   .map(IlpOverHttpLinkSettings.AuthType::valueOf)
-                  .ifPresent(builder::authType);
+                  .ifPresent((authType) -> {
```

Comment:
Minor nit, but the standard `continuation-indent` is 4-spaces for Quilt. This is a bit different from the Connector, but we should keep with the existing style for now.

Reply:
Fixed via 678b93bdb900cd4c22cb8b14ea7d71e4ef876ccd